src/lib.rs

```diff
@@ -125,6 +126,7 @@ pub use scope::EscapableHandleScope;
 pub use scope::HandleScope;
 pub use scope::TryCatch;
 pub use script::ScriptOrigin;
+pub use script_compiler::CachedData;
```

Comment:
BufferPolicy should also be exported?

Maybe rename to CachedDataBufferPolicy?

Reply:
The current `CachedData` API doesn't support owned buffers so `BufferPolicy` isn't used. Owned buffers are freed by C++ destructor and would need extra wrappers if using them from the Rust side is needed.

Should it still be exported if unused?